src/interfaces/iPeer.ts

```diff
@@ -0,0 +1,39 @@
+import { Message } from "../models/message";
+
+/**
+ * Peer interface.
+ */
```

Comment:
Useless comment, the name IPeer already implies that this is a Peer Interface.

Reply:
Added for consistency, for when generating TSDoc for example. This is also done in for example blocker-models.